systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeCors.java

```diff
@@ -59,7 +53,7 @@ void testCorsOriginAllowed(VertxTestContext context) {
         JsonObject topics = new JsonObject();
         topics.put("topics", topic);
 
-        client.request(HttpMethod.OPTIONS, bridgePort, bridgeHost, "/consumers/" + groupId + "/instances/" + kafkaBridgeUser + "/subscription")
+        client.request(HttpMethod.OPTIONS, bridgeHost, "/consumers/" + groupId + "/instances/" + kafkaBridgeUser + "/subscription")
```

Comment:
why the choice of makingthe host an explicit parameter while hiding the port (80) inside the method?

Reply:
Yeah sorry, I leave that after some try, removed :)